apps/studio/src/components/CoreTabs.vue

```diff
@@ -635,7 +613,15 @@ export default Vue.extend({
         }
       }
     },
-    async close(tab) {
+    async close(tab: OpenTab) {
+      if (tab.unsavedchanges) {
+        const confirmed = await this.$confirm(
+          `really close [icon here] ${tab.title}?`,
```

Comment:
looks like this has a placeholder? `[icon here]`?

Reply:
Lol, I forgot. My bad.

Reply:
Once this is done, I'll get it merged it. Probably for the next update

Reply:
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/38707148/647eff89-49eb-4b73-af45-bc312f3211a6)

done :)